### DISCUSSION TITLE
请问如果一部影片分上下或者多集应该怎么命名

### DISCUSSION THREAD

**Discussion by ybjopq007** [metatube-community/jellyfin-plugin-metatube#349] [Q&A]
wiki里的Naming rules (命名规范) 空白页 没有注释 请问应该如何命名 比如一部影片较大 片源分为多部发布 _1 _2 _3这种应该怎么操作 现在直接搜刮出多个一样的

**Comment by YuleLee-652987** [comment]
可以多选选中这几个影片，然后点击整合版本（EMBY是这样操作的）。

**Comment by xianyunyeheA** [comment]
https://raw.githubusercontent.com/metatube-community/metatube-community.github.io/main/docs/wiki/naming-rules.md
